app/modules/pin/pin.route.ts

```diff
@@ -0,0 +1,25 @@
+import PinController from './pin.controller';
+import { IRoute, IRouteConfig, HttpMethods, AuthStrategies } from '../../interfaces/utils/Route';
+
+class PinRoute implements IRoute {
+  public basePath = '/pins';
+  public controller = new PinController();
+
+  public getServerRoutes(): IRouteConfig[] {
+    return [
+      {
+        method: HttpMethods.POST,
+        auth: AuthStrategies.PUBLIC,
```

Comment:
@abdullahtariq91 ask Nestor if it should be OAUTH or JWT auth. As maybe it doesn't makes sense to leave it public?